clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/ClusterCleanupAgent.java

```diff
@@ -0,0 +1,71 @@
+package com.netflix.spinnaker.clouddriver.titus.caching.agents;
+
+import static com.netflix.spinnaker.cats.agent.AgentDataType.Authority.INFORMATIVE;
+import static com.netflix.spinnaker.clouddriver.core.provider.agent.Namespace.*;
+
+import com.netflix.spinnaker.cats.agent.AgentDataType;
+import com.netflix.spinnaker.cats.agent.CacheResult;
+import com.netflix.spinnaker.cats.agent.CachingAgent;
+import com.netflix.spinnaker.cats.agent.DefaultCacheResult;
+import com.netflix.spinnaker.cats.provider.ProviderCache;
+import com.netflix.spinnaker.clouddriver.titus.TitusCloudProvider;
+import com.netflix.spinnaker.clouddriver.titus.caching.Keys;
+import com.netflix.spinnaker.clouddriver.titus.caching.TitusCachingProvider;
+import com.netflix.spinnaker.clouddriver.titus.caching.utils.CachingSchema;
+import java.util.*;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ClusterCleanupAgent implements CachingAgent {
```

Comment:
This is super similar to the ClusterCleanupAgent in aws - I could probably not have these basically be C&P of each other